src/Api/Serializer/FlagSerializer.php

```diff
@@ -32,6 +32,7 @@ protected function getDefaultAttributes($flag)
             'reason'       => $flag->reason,
             'reasonDetail' => $flag->reason_detail,
             'createdAt'    => $this->formatDate($flag->created_at),
+            'createdAtDisplay' => date_format($flag->created_at, 'd/m/Y H:i:s'),
```

Comment:
Please do this in JS code, as we do in other places. There are helpers like `humanTime` that could be used.